test/testHelpers.hpp

```diff
@@ -557,64 +587,81 @@ ::testing::AssertionResult elemWiseEq(std::string aName, std::string bName,
         return ::testing::AssertionSuccess();
     } else {
         int idx = std::distance(b.begin(), bItr);
+        af::dim4 aStrides = calcStrides(aDims);
         af::dim4 bStrides = calcStrides(bDims);
         af::dim4 coords = unravelIdx(idx, bDims, bStrides);
 
-        return ::testing::AssertionFailure() << "VALUE DIFFERS:\n"
-                                             << " at ([" << coords << "]):\n"
-                                             << aName << "(" << a[idx] << ")\n"
-                                             << bName << "(" << b[idx] << ")";
+        return ::testing::AssertionFailure()
+            << "VALUE DIFFERS:\n"
+            << " at ([" << coords << "]):\n"
+            << aName << "(" << a[idx] << ")\n"
```

Comment:
Should probably combine the context and the invalid value since they show the same information.